pkg/spec/spec.go

```diff
@@ -469,10 +465,15 @@ func blockAccessToKernelFilesystems(config *CreateConfig, g *generate.Generator)
 			"/proc/sched_debug",
 			"/proc/scsi",
 			"/sys/firmware",
+			"/sys/fs/selinux",
 		} {
 			g.AddLinuxMaskedPaths(mp)
 		}
 
+		if config.PidMode.IsHost() && rootless.IsRootless() {
```

Comment:
Why return here instead of adding the read-only paths?

Reply:
Because these are not allowed if you are sharing PIDs.   I moved this check down.
When using --pid=host ,the readonly bind mounts fail.